e2e_test/e2e_test.go

```diff
@@ -468,6 +507,50 @@ var _ = g.Describe("E2E integration testing", g.Ordered, func() {
 				}, timeout, interval).Should(BeTrue())
 			})
 		})
+
+		g.Context("With more ServiceFunctionChains than resources", g.Ordered, func() {
+			g.It("Should have all SFC Pods eventually running", func() {
+
+				dpuResNum := getMaxDpuResources(dpuSideClient)
+				Expect(dpuResNum).To(BeNumerically(">=", 0))
+
+				sfcNum := (dpuResNum / 2) + 2
+				if sfcNum > sfcNumMax {
+					sfcNum = sfcNumMax
+				}
+
+				g.By(fmt.Sprintf("Creating %d SFCs to allocate all DPU side resources", sfcNum))
+				for i := 0; i < sfcNum; i++ {
+					sfc := sfcNew(i)
+					testutils.SfcCreate(dpuSideClient, sfc)
+
+					Eventually(func() bool {
+						return testutils.GetPod(dpuSideClient, sfc.Spec.NetworkFunctions[0].Name, sfc.ObjectMeta.Namespace) != nil
+					}, timeout, interval).Should(BeTrue())
+				}
+
+				for i := 0; i < sfcNum; i++ {
+					g.By(fmt.Sprintf("Find a running SFC Pod (%d of %d)", i, sfcNum))
+					sfc := sfcFindRunningSfc(dpuSideClient, sfcNum)
```

Comment:
This is flaky because above when you create the SFC, the scheduler doesn't necessarily follow FIFO ordering.

Reply:
`sfcFindRunningSfc()` returns any running SFC that it finds. It does not rely on any particular ordering in which the scheduler starts the pods.